src/amber/cli/commands/database.cr

```diff
@@ -52,17 +52,17 @@ module Amber::CLI
             `crystal db/seeds.cr`
             Micrate.logger.info "Seeded database"
           when "migrate"
-            Micrate::Cli.run_up
+            begin
+              Micrate::Cli.run_up
+            rescue e : IndexError
+              exit! "no migrations to run"
```

Comment:
Just capitalize `"No migrations to run."`

Maybe we should state something more specific as the error `"No migrations to run in #{MIGRATIONS_DIR}."`